_drafts/CocoaPods-Stats.markdown

```diff
@@ -0,0 +1,99 @@
+---
+layout: post
+title:  "CocoaPods Stats"
+author: orta
+categories: cocoapods.org release stats
+---
+
+People have been asking for years about getting feedback on how many downloads their libraries have got. We've been thinking about the problem for a while, and finally ended up asking Segment if they would provide a backend for the project.
+
+<!-- more -->
+
+### But wait, there's more...
+
+It wasn't just enough to offer just download counts. We spend a lot of time working around Xcode's project file intricacies, however in this context, it provides us with foundations for a really nice feature. CocoaPods Stats will be able to keep track of the unique number of installs within Apps / Watch Apps / Extensions / Unit Tests.
+
+What this means is that developers using continuous integration only increase the downloads of the Pod, but register as 1 install. Separating total installations vs actual downloads.
```

Comment:
This means is that developers using continuous integration only register as 1 install, even if the pod gets downloaded each time, separating total installations vs actual downloads.